keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/README.md

```diff
@@ -650,12 +654,15 @@ TODO
   - 99% percentile over day of absolute value from (job creation time minus expected
     job creation time) for cron job <= 10%
   - 99,9% of /health requests per day finish with 200 code
+N/A
 
 * **Are there any missing metrics that would be useful to have to improve
   observability if this feature?**
   Describe the metrics themselves and the reason they weren't added (e.g. cost,
   implementation difficulties, etc.).
 
+Pods using this feature are required to set a new field, which increases the size of their objects.
```

Comment:
This is misplaced, right?  This should be scalability section for `Will enabling / using this feature result in increasing size or count of the existing API objects?` question.